native/react-version-test.js

```diff
@@ -0,0 +1,23 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+it('native/* should be ^16.0.0', () => {
```

Comment:
Should we keep these?

Reply:
So I pondered this too ... whilst the shrinkwrap determines what version we use, it does not guarantee that node resolves the right version. I kinda like them. They are cheap and will act as a failsafe.

Reply:
That shounds good to me :)